docker-image/Dockerfile.amd64

```diff
@@ -1,8 +1,13 @@
-FROM alpine:3.8 as base
+FROM debian:9.8-slim as base
 MAINTAINER Shaun Crampton <shaun@tigera.io>
 
 # Since our binary isn't designed to run as PID 1, run it via the tini init daemon.
-RUN apk add --update tini
+ENV TINI_VERSION v0.18.0
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /sbin/tini
```

Comment:
there's no version online that's statically built?

then we could skip `musl`

Reply:
This is in fact the statically built option: 
https://github.com/krallin/tini#statically-linked-version

I just assumed the original way of installing `tini` using `apk add` was also using a statically built version. So I wasn't sure about removing `musl`. 

Looking at the [original commit when this was added](https://github.com/projectcalico/typha/commit/182918d658e3f0f349dc99016891e6115e91d00a), it definitely looked like `musl` was added for `tini`. But I just wasn't sure if it was still needed. 

I can just remove `musl` and see if the build still works.

cc @fasaxc any thoughts on this?

Reply:
Just removed the `musl` statements and tests for arch `amd64` passed with no issues. 

I'll take it out then.

Reply:
awesome :-)

Reply:
This has been resolved now. Using statically built version and removed `musl` dependency.